src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/AddTeams.js

```diff
@@ -168,13 +168,15 @@ $(document).ready(function () {
         }
 
         privates.getAllGroupsOptions(function (options) {
-
-            var responseOptions = "<option value = '0'>" + currNs.groupIsNotSelectedMessage + "</option>";
-
-            $.each(options, function (key, value) {
-                responseOptions += "<option value='" + value.Id + "'>" + value.Name + "</option>";
-            });
-       
+            if (options.length == 1) {
+                var responseOptions = "<option value ='" + options[0].Id + "'>" + options[0].Name + "</option>";
```

Comment:
Can you still execute this line inside for each? Given there is just one item it will do the same. Then you don't need if probably

Reply:
but we decided that it is better for UI. because when we automatically use the first group, it is ununderstanding that it is can to be more than that 1 group.
or do you mean do not use "var"?

Reply:
I do not mean to change functionality. In terms of issue everything is good.

Think about a ways how to write less repetitive code. Right now lines 172 and 175 do the same thing.

Reply:
no, it another thing, in 172 we choice out 1 group
in 175 we choice "no group selected" 
![image](https://user-images.githubusercontent.com/33187342/36151866-ad99a5a6-10d1-11e8-8d27-1aec060c9c80.png)

Reply:
I'm referring to line 175 only without anything around

Reply:
we do not understand, can you explain more detail, please?

Reply:
How can this task be a QA? or do you mean that we need to make some QA?